integrations/integration.php

```diff
@@ -63,10 +63,17 @@ public function __construct( string $slug ) {
 	 * @private
 	 */
 	public function activate( array $options = [] ): void {
+		// If integration is already available in customer code then don't activate it from platform side.
+		if ( $this->is_loaded() ) {
+			trigger_error( // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_trigger_error
+				sprintf( 'Prevented activating of integration with slug "%s" because it is already available via customer code.', esc_html( $this->slug ) ),
```

Comment:
```suggestion
				sprintf( 'Prevented activating of integration with slug "%s" because it is already loaded.', esc_html( $this->slug ) ),
```

Reply:
Addressed in https://github.com/Automattic/vip-go-mu-plugins/pull/4839/commits/4bc3fdeda3547e91dcbc6c8656666e72049b7e2f